ConsoleApp1/Weather.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1
+{
+    internal class Weather
+    {
+        [JsonProperty("location")]
+        private Results location;
+
+        [JsonProperty("current")]
+        private Current current;
+     
+        private void PrintWeather()
+        {
+            location.LocationInfo();
+            current.CurrentWeatherInfo();
+        }
+
+        private void GetWeatherForecast(string city)
+        {
+            HttpClient client = new HttpClient();
+
+            string apiUrl = $"http://api.weatherstack.com/current?access_key=038f58b1cb7e69df48eed90066f68322&query={city}";
+
+            var result = client.GetAsync(apiUrl).Result.Content.ReadAsStringAsync().Result;
```

Comment:
`.Result ` is a blocking operation so you are blocking the thread until there will be the result, use `async-await`

Reply:
you can call `GetStringAsync` directly from `httpClient`